build_projects/dotnet-cli-build/DebTargets.cs

```diff
@@ -28,33 +28,36 @@ public static BuildTargetResult GenerateSdkDeb(BuildTargetContext c)
             var version = c.BuildContext.Get<BuildVersion>("BuildVersion").NuGetVersion;
             var debFile = c.BuildContext.Get<string>("SdkInstallerFile");
             var manPagesDir = Path.Combine(Dirs.RepoRoot, "Documentation", "manpages");
-            var previousVersionURL = $"https://dotnetcli.blob.core.windows.net/dotnet/{channel}/Installers/Latest/dotnet-ubuntu-x64.latest.deb";
             var sdkPublishRoot = c.BuildContext.Get<string>("CLISDKRoot");
             var sharedFxDebianPackageName = Monikers.GetDebianSharedFrameworkPackageName(CliDependencyVersions.SharedFrameworkVersion);
+            var debianConfigFile = Path.Combine(Dirs.DebPackagingConfig, "dotnet-debian_config.json");
+            var postinstallFile = Path.Combine(Dirs.DebPackagingConfig, "postinst");
 
-            var objRoot = Path.Combine(Dirs.Output, "obj", "debian", "sdk");
-
-            if (Directory.Exists(objRoot))
+            var debianConfigVariables = new Dictionary<string, string>()
             {
-                Directory.Delete(objRoot, true);
-            }
+                { "SHARED_FRAMEWORK_DEBIAN_PACKAGE_NAME", sharedFxDebianPackageName },
+                { "SHARED_FRAMEWORK_NUGET_NAME", Monikers.SharedFrameworkName },
+                { "SHARED_FRAMEWORK_NUGET_VERSION",  CliDependencyVersions.SharedFrameworkVersion },
+                { "SHARED_FRAMEWORK_BRAND_NAME", Monikers.SharedFxBrandName },
```

Comment:
Are all these "SHARED_FRAMEWORK" variables being used in the config.json file?  I only see `SHARED_FRAMEWORK_DEBIAN_PACKAGE_NAME` used.